modules/deploy/pages/deployment-option/cloud/serverless.adoc

```diff
@@ -56,67 +35,53 @@ Each Serverless cluster has the following limits:
 
-== Free trial 
-
-Start using Redpanda Serverless beta by joining the free trial. To ask questions, give feedback, or extend your trial, use the *#serverless* https://redpandacommunity.slack.com/[Community Slack^] channel, or contact https://support.redpanda.com/hc/en-us/requests/new[support^].
-
-=== Prerequisites
+== Trial credits
 
-. https://redpanda.com/try-redpanda?section=cloud-trial[Sign up for a beta trial^] of Redpanda Serverless.
+Redpanda issues 100 free credits, which is enough to run Redpanda with reasonable throughput. One credit equals one US dollar (USD). These credits expire in 14 days. Usage is based on the following: 
 
-. Make sure you have the latest version of `rpk`. See xref:get-started:rpk-install.adoc[].
-
-=== Credits and limits
-
-Each beta trial supports five Serverless clusters. Redpanda issues 100 free credits, which is enough to run Redpanda with reasonable throughput. These credits expire in 14 days. Usage is based on the following:
-
-* Ingress: 0.1 credit per GB in (writes)
-* Egress: 0.1 credit per GB out (reads)
-* Partitions: 0.003 credit (metered in 10 min fractions) per partition/hour	
-* Storage: 0.1 credit (metered in 10 min fractions) per GB stored/month
-
-After either the 14 day trial expires or the 100 credits expire, the clusters move into a suspended state, and you won't be able to access your data in either the Redpanda Cloud UI or with the Kafka API. There is a 14 day grace period following the end of the trial, in case you want to extend it. After that, the data is permanently deleted. To extend your trial, use the *#serverless* https://redpandacommunity.slack.com/[Community Slack^] channel, or contact https://support.redpanda.com/hc/en-us/requests/new[support^].
-
-=== Get started
+* Ingress: 0.1 credit per GB of data in written into Redpanda
+* Egress: 0.1 credit per GB of data read out of Redpanda
+* Partitions: 0.003 credit (metered in 10 minute fractions) per partition/hour	
+* Storage: 0.1 credit (metered in 10 minute fractions) per GB stored/month
 
-For the beta release, you'll receive an email to activate your Serverless trial. 
+After either the 14 day trial expires or the 100 credits expire, the clusters move into a suspended state, and you won't be able to access your data in either the Redpanda Cloud UI or with the Kafka API. There is a 14 day grace period following the end of the trial. After that, the data is permanently deleted. 
 
-. In your Welcome email, click to sign up.
-+
-Redpanda prepares a Redpanda Cloud account for you with a "Hello, World!" Serverless cluster in AWS.
+== Get started
 
-. To log in to Redpanda Cloud, run the following command in your terminal:
-+
-```
-rpk cloud login
-```
+Redpanda prepares an account for you within one minute of starting the trial. 
```

Comment:
Why do we need to say "one minute" exactly? It's the minimum amount of time required to spin the cluster? If it's not a hardcoded time, we should be vague here.

Reply:
@vladoschreiner should we say under one minute? Or instantly?

Reply:
changed to instantly, to match Create cluster page in UI